[26.03] Docs: FlashInfer release (#201)
ROCm/flashinfer: amd-docs/examples/flashinfer-examples.rst

.. meta::
  :description: FlashInfer examples
  :keywords: FlashInfer, programming, ROCm, example, sample, tutorial

.. _run-a-flashinfer-example:

********************************************************************
Run a FlashInfer example
********************************************************************

> The examples folder in the `https://github.com/ROCm/flashinfer/tree/amd-integration/examples <https://github.com/ROCm/flashinfer/tree/amd-integration/examples>`__ repository has example code that you can run FlashInfer with.

Reviewer: ```suggestion
The examples folder in the `https://github.com/ROCm/flashinfer/tree/amd-integration/examples <https://github.com/ROCm/flashinfer/tree/amd-integration/examples>`__ repository has example code that you can use to run FlashInfer.
```

> You can save the following code snippet to a Python script once you have FlashInfer installed and run the script to try it out.

Reviewer: ```suggestion
You can save the following code snippet to a Python script after you have FlashInfer installed, then run the script to try it out.
```
